src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = String.valueOf(colorSupplier.getRandomColor());
+        int randomNumber = new Random().nextInt(100) + 1;

```

Comment:
Random object creates every time you call getRandomColor. Think of the approach, where you create an object of Random only once for this class